client/src/components/categorical/value/index.js

```diff
@@ -677,8 +677,8 @@ class CategoryValue extends React.Component {
               display={isColorBy && categoryValueIndices ? "auto" : "none"}
               style={{
                 marginLeft: 5,
-                width: VALUE_HEIGHT,
-                height: VALUE_HEIGHT,
+                width: 15,
```

Comment:
Can we set the `VALUE_HEIGHT` variable directly instead of hardcoding?

Reply:
Renaming it. It's actually stacked bar height, and it relates to value because of Roboto Condensed at 11 pt's height. The engineering work to make that pixel perfect would be fragile.

![image](https://user-images.githubusercontent.com/1770265/111229689-b875b600-85bc-11eb-8f8c-64299b732e98.png)

Reply:
How about adding a new global then?  Maybe called `STACKED_BAR_HEIGHT`?